clean_code/player.py

```diff
@@ -1,142 +1,153 @@
-            song = EasyID3(item)
-            song_data = (str(key + 1) + ' : ' + song['title'][0] + ' - '
-                         + song['artist'][0])
-            text1.insert(END, song_data + '\n')
-    except:
-        pass
-#################################################################################
-def song_data():
-    global list_index, list1, SONG_END, pausing, label1, text1
-    try:
-        song = EasyID3(list1[list_index])
-        song_data = "Now playing: Nr:" + str(list_index + 1) + " " + \
-                    str(song['title']) + " - " + str(song['artist'])
-        return song_data
-    except:
-        pass
-#################################################################################
-def button2():
-    global list_index, list1, SONG_END, pausing, label1, text1
-    try:
-        directory = list1[list_index]
-        pygame.mixer.music.load(directory)
-        pygame.mixer.music.play(1, 0.0)
-        pygame.mixer.music.set_endevent(SONG_END)
-        pausing = False
-        label1['text'] = song_data()
-    except:
-        pass
-#################################################################################
-def check_music():
-    global list_index, list1, SONG_END, pausing, label1, text1
-    try:
-        for event in pygame.event.get():
-            if event.type == SONG_END:
-                button5()
-    except:
-        pass
-#################################################################################
-def button3():
-    global list_index, list1, SONG_END, pausing, label1, text1
-    try:
-        if pausing:
+
+        self.list_of_songs = []
+        self.paused = False
+        self.current_song_index = 0
+
+        self.song_info_label = None
+        self.song_list_text = None
+
+        self.create_widgets()
+
+    def create_widgets(self):
+        self.create_buttons()
+        self.create_song_info_label()
+        self.create_song_list_text()
+
+    def create_buttons(self):
+        BUTTONS = [
+            ("ADD TO LIST", self.add_songs_to_list, self.BUTTONS_ROW[0]),
+            ("PLAY SONG", self.play_song, self.BUTTONS_ROW[1]),
+            ("PAUSE/UNPAUSE", self.toggle_pause, self.BUTTONS_ROW[2]),
+            ("PREVIOUS SONG", self.play_previous_song, self.BUTTONS_ROW[3]),
+            ("NEXT SONG", self.play_next_song, self.BUTTONS_ROW[4]),
+        ]
+
+        for text, command, row in BUTTONS:
+            self.create_button(text, command, row)
+
+    def create_button(self, text, command, row):
+        Button(self, text=text, command=command, bg=self.BUTTON_BG_COLOR, width=self.BUTTON_WIDTH).grid(row=row, column=self.COLUMN)
+
+    def create_song_info_label(self):
+        self.song_info_label = Label(self, fg='black', font=self.FONT_STYLE, bg=self.LABEL_BG_COLOR)
+        self.song_info_label.grid(row=self.SONG_INFO_ROW, column=self.COLUMN)
+
+    def create_song_list_text(self):
+        self.song_list_text = Text(self, wrap=WORD, width=self.TEXT_WIDTH)
```

Comment:
When using variables / functions from a package, it is better to explicilty specify that they come from that package inline. Here for example:

```suggestion
        self.song_list_text = tkinter.Text(self, wrap=tkinter.WORD, width=self.TEXT_WIDTH)
```